src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,45 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Foo {
+ *  protected int f;
+ *
+ *  public Foo() {};
+ *  public Foo(int f) {
+ *     this.f = f;
+ *  }
+ *  public Object clone() { // violation, does not call super.clone()
+ *    Foo other = new Foo();
+ *    other.f = this.f;
+ *    return other;
+ *  }
+ * }
+ *
+ * class FooA extends Foo {
```

Comment:
Please extend some class from jdk, Check do not care about any hierarchy, we should not give use false feeling that consider hierarchy. All classes have such cline method to override.